README.md

```diff
@@ -111,6 +111,107 @@ This repository also contains the tooling required to install Foreman Katello.
 
 See [KATELLO.md](/KATELLO.md) for details and specifics of that setup.
 
+# Certificates
+
+## Terminology
+
+* **default CA** - a CA generated by the installer, and used to generate server and client certificates. This CA is used by Candlepin to generate client certificates.
+* **server CA** - a CA that can be provided by the user, or is a copy of the default CA, used by all public facing interfaces
```

Comment:
This is really Katello terminology that we don't use for Foreman. I'd be explicit in that.